tests/unittests/conftest.py

```
@@ -0,0 +1,10 @@
+# tests/unit/conftest.py
+
+import pytest
+
+def pytest_collection_modifyitems(config: pytest.Config, items: list[pytest.Item]) -> None:
+    """Automatically mark all tests collected in this directory as 'unit' tests."""
+    from rich import print
```

Ruff (0.8.2)

`rich.print` imported but unused

Remove unused import: `rich.print`

(F401)